test/framework/config/core_test.clj

```diff
@@ -0,0 +1,12 @@
+(ns framework.config.core-test
+  (:require
+   [clojure.test :refer :all]
+   [framework.config.core :as config]))
+
+(def config-map (config/read-edn-file nil))
+
+(deftest its-key-present
```

Comment:
Small nitpick - test fns should be 'is-key-present' and 'is-key-not-present'

Reply:
You right, @juan-ignacio-848 also point that out.
 
I'll change that.

Reply:
No, I don't think so. They should be like, `key-present?`.

Please, refer https://github.com/bbatsov/clojure-style-guide#naming-predicates for further readings.

Reply:
Here: https://github.com/Flexiana/framework/pull/87/commits/c07fbb36529c5f397a20285a90a8e2306b2fd18d